### ISSUE TITLE
Why are the support forums down?

### ISSUE THREAD

**Issue by dabockster** [ci-bonfire/Bonfire#1274]
Question is in the title. I wanted to read some user documentation, but the link to the forums on the Bonfire website gives a 403 error.

**Comment by soujux** [comment]
I would like you to reopen the forum, in order to receive collaborations from the comunity.  This is a great framework

**Comment by lonnieezell** [comment]
I'll consider it again. I closed them down because I was getting very few participants anymore on the forums, and would spend more time fighting spam than was useful.

**Comment by dabockster** [comment]
I understand the war against spammers. Would it be possible to reopen them in a read-only mode with a notice along the lines of "All future community support is located [insert url here]"?

**Comment by lonnieezell** [comment]
The biggest problem is that about 1.5 years ago my host's server had a freak out and lost all of my websites. Unfortunately, I didn't have a current backup that wasn't through them and, after much arguing with them, I left them. I had a PHPBB-based one up for a bit after that which I had other issues with - and not a whole lot of visitors, anyway. 

So, unfortunately, that old data is lost and I can't do a read-only version. Which stinks. Your post did inspire, me, though, and I'm currently [building out a simple one](https://github.com/lonnieezell/simple-forums) in CodeIgniter 4. This works as both a trial for me to use and test for pain points in CI4 while I'm developing it, and will work as some forums for Bonfire.

**Comment by dabockster** [comment]
Sorry to hear about your hosting problems. I've had similar issues in the past with various webhosts.

Nice to hear that my post inspired you a bit. Hopefully you can use this inspiration to get something up that's a bit more centralized.

If you do find any of the old forum data anywhere, let me know.

**Comment by Reconix** [comment]
They're up again.

http://bonfire-forums.net

**Comment by dabockster** [comment]
@Reconix Awesome news! Still doesn't recover the knowledge lost from the old forum, but it's a good step in the right direction.

**Comment by Reconix** [comment]
True, will see if I can set some Archive Forum up maybe try and pull the most important posts from the past.
Been really busy lately but hold tight, will put it on the todo list.
